dissect/target/plugins/os/windows/wer.py

```diff
@@ -18,7 +19,16 @@ def _collect_wer_data(wer_file: Path) -> tuple[list[tuple[str, str]], dict[str,
     record_values = {}
     record_fields = []
     key = None
-    for line in wer_file.read_text("utf-16").splitlines():
+
+    # Default encoding, when no BOM is present
+    encoding = "utf-16-le"
+
+    # Check for the presence of a BOM header and use the correct encoding
+    with wer_file.open("rb") as fh:
+        if fh.read(len(codecs.BOM)) == codecs.BOM_UTF16_LE:
```

Comment:
```suggestion
        if fh.read(len(codecs.BOM)) in (codecs.BOM_UTF16_LE, codecs.BOM_UTF16_BE):
```
Or is is guaranteed to be LE?